tools/GraphClust/Preprocessing/splitSHAPE.py

```diff
@@ -36,7 +36,7 @@
     lines = content.split('\n')
     for line in lines:
         if pattern.match(line):
-            line = line.replace('>','').strip()
+            line = line.replace('>','').split()[0]
```

Comment:
Before it was assigning the full line now only the first element. I guess first element of the split is independent of striping, right?